samples/GraphQL.DataLoader.Example/GraphQl/CarsGraphType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using DataLoaderGql.Types;
+using GraphQL.Types;
+
+namespace DataLoaderGql.GraphQl;
+
+public class CarsGraphType : ObjectGraphType<Car>
+{
+    public CarsGraphType()
+    {
+        Field(x => x.Id, type: typeof(IntGraphType)).Description("Id of the car");
+        Field(x => x.Model, type: typeof(StringGraphType)).Description("Model of the car");
+        Field(x => x.Price, type: typeof(IntGraphType)).Description("Price of the car");
+        Field(x => x.ModelYear, type: typeof(IntGraphType)).Description("Model year of the car");
+    }
+}
```

Comment:
files must end with newline.  note: `dotnet format` should fix a lot of these formatting errors, but tests were broken and didn't run.  see #4101

Reply:
do i need something special up? dotnet format on the csproj doesnt change anything when i run it.

Reply:
i think it does end with a newline now tho